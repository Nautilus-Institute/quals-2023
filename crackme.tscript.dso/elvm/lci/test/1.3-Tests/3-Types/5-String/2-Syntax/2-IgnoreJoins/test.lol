HAI 1.3
	VISIBLE "Lorem ipsum, VISIBLE dolor sit"
KTHXBYE
