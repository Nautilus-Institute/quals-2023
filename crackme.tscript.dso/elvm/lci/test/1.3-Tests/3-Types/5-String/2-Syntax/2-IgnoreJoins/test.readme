This test checks that strings ignore joining lines with the comma.
