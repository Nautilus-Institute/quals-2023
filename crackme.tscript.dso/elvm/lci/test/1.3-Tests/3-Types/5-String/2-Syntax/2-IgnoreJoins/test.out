Lorem ipsum, VISIBLE dolor sit
